backends/platform/atari/readme.txt

```diff
@@ -0,0 +1,422 @@
+ScummVM
+=======
+
+This is a new port of ScummVM (https://www.scummvm.org), a program which allows
+you to run certain classic graphical adventure and role-playing games, provided
+you already have their data files.
+
+You can find a full list with details on which games are supported and how well
+on the compatibility page: https://www.scummvm.org/compatibility.
+
+
+Yet another port?
+-----------------
+
+Yes, I am aware of the official Atari/FreeMiNT port done by KeithS over the
+years (https://docs.scummvm.org/en/v2.6.1/other_platforms/atari.html). It is
+even updated every release and put on the official ScummVM website. That port
+is basically just a recompiled SDL backend for our platform - that has some
+advantages (works in GEM, can be easily compiled for the FireBee etc) but also
+a few disadvantages, like:
```

Comment:
Could you please flip it over and explain in what areas this port is different/more suitable for the low-power platform?

I'll put couple of examples below.

Reply:
Your examples are more than fitting so I used them nearly without any modification. I have also flipped the paragraph above over.